modules.d/90crypt/module-setup.sh

```diff
@@ -96,8 +96,12 @@ install() {
     fi
 
     inst_hook cmdline 30 "$moddir/parse-crypt.sh"
+    inst_hook shutdown 24 "$moddir/crypt-shutdown.sh"
```

Comment:
Yes, correct. crypt-shutdown needs to be called before dm dracut module, since the crypt device mappings we reference would otherwise no longer exist.